Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM centos:centos7.5.1804
+
+MAINTAINER tangxiaoqing214445
+
+ENV DEFAULT_DIR /var/local
+
+# add code repository
+ADD thirdparty /var/local/thirdparty
+
```

Comment:
You should add palo source code in docker image too, otherwise you won't build palo.

Reply:
OK